azavea/raster-vision
# Factor-out Commonalities into Base Image

## Overview

Further optimize the docker build.  All commonalities are now in a common ancestor image.

To test:
```
./docker/build
touch scripts/rastervision
./docker build
```

The second `docker/build` should happen much more quickly.  Tensorflow should not be reinstalled.

### Checklist


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as rebuilding the Docker image.
* Include test case, and expected output if not captured by automated tests.
